doc/redirects.json

```diff
@@ -3724,15 +3724,6 @@
   "sec-language-texlive-lualatex-font-cache": [
     "index.html#sec-language-texlive-lualatex-font-cache"
   ],
-  "titanium": [
-    "index.html#titanium"
-  ],
-  "building-a-titanium-app": [
-    "index.html#building-a-titanium-app"
-  ],
-  "emulating-or-simulating-the-app": [
-    "index.html#emulating-or-simulating-the-app"
-  ],
```

Comment:
So *actually* the point of these redirect mappings is to redirect what moved around. In case we're deleting identifiers, those locations should redirect to a tombstone, such as the release notes explaining what happened. 

Therefore just deleting the mappings is very much undesireable -- how else are users supposed to find out what happened when their bookmark or some link they followed gets them nowhere? We already had two decades of manually searching for keywords, let's put an end to this.

Now the problem is that for some, to me inexplicable, reason we're lumping together Nixpkgs and NixOS release notes. The way nixos-render-docs is built and used we (rightfully) can't check validity of items in an entirely different document. So there's no clean way out of this for the time being.

IMHO the correct thing to do is open a release notes section for Nixpkgs, import it from the NixOS manual for completeness, and eventually enforce its use via documentation rendering tools. 

(@hsjobeki @erictapen cf. systemic confusion around the semantics of objects of interest and component boundaries)

Reply:
Thanks @DimitarNestorov for the addition. I failed to note that this is a non-blocking comment, and I deem it the documentation team's responsibility to provide the infrastructure and guidance so contributors don't run into that sort of friction. But adding the new section is the ideal outcome in my view, so thanks again.